modules/gce_worker/main.tf

```diff
@@ -23,6 +31,18 @@ variable "worker_image" {
   default = "ubuntu-os-cloud/ubuntu-1804-lts"
 }
 
+variable "worker_service_accounts_count_com" {
+  default = 4
```

Comment:
any reason for 4 as default?  Just a number to begin with?

Reply:
My thinking was that it's the default `length(var.zones)` :man_shrugging: